keyword_match/keyword_matcher.py

```diff
@@ -29,7 +18,7 @@ def fetchCSVFromGit(url, index_col):
 * Reference: http://jonathansoma.com/lede/algorithms-2017/classes/fuzziness-matplotlib/fuzzing-matching-in-pandas-with-fuzzywuzzy/
 """
 
-def fuzzyScore(query, candidate_sentences, limit, scoring_method='ratio'):
+def _fuzzyScore(query, candidate_sentences, limit, scoring_method='ratio'):
```

Comment:
Methods with leading underscores are they private methods? 
If so i think the python standard is to use two of them Ex: `def __fuzzyScore`

Reply:
Oh I didn't know we can exchange comments on GitHub. Yes, _method is a private method. I looked up, but I think the standard is one underbar. Either way is fine, but we gotta stick with one format.

Reply:
https://docs.python.org/3/tutorial/classes.html#private-variables
check out this link it says 
> Any identifier of the form __spam (at least two leading underscores, at most one trailing underscore) is textually replaced with _classname__spam

its only 1 trailing underscore not the leading

Reply:
I see. Yeah, you're right. For private methods, the name should be double underscore. Actually, this is not a class, so I'd call it a helper function, which usually starts with one underscore.

Reply:
That being said, do you wanna make keyword matcher a class? The difference would be something like you don't have to pass a searching space to matching function every time user invokes the function. It depends on how you wanna implement the function on application side.